src/lib/sql/sql_pipeline.hpp

```diff
@@ -47,7 +49,7 @@ class SQLPipeline : public Noncopyable {
   const SQLQueryPlan& get_query_plan();
 
   // Returns all tasks tht need to be executed for this query.
```

Comment:
Returns all tasks **that** ...

Reply:
Also I would update the comment to reflect that we return task sets instead of tasks now.